tests/testconfig.ttl

```diff
@@ -342,8 +342,7 @@
           rdfs:label "Caption"@en, "Luokka"@fi ],
         [ skosmos:property skos:notation ;
           rdfs:label "UDC number"@en, "UDC-numero"@fi, "UDC-nummer"@sv ;
-          rdfs:comment "Class Number"@en ] ;
-	skosmos:sparqlGraph <http://www.skosmos.skos/conceptPropertyLabels/> .
```

Comment:
Why did you remove this setting? (I'm sure it's OK to remove, I'd just like to understand the reason)

Reply:
I still think another solution is needed here. I can take a look...

Reply:
Solved.